homeassistant/components/zwave_js/update.py

```diff
@@ -244,6 +246,8 @@ async def async_install(
 
             # We need to block until we receive the `firmware update finished` event
             await self._finished_event.wait()
+            # Clear the event so that a second firmware update blocks again
```

Comment:
We currently only subscribe for a single event with `node.once`. We need to change that to make this work.

Reply:
good catch. It still hangs after the above though

Reply:
issue was that we weren't giving enough time for the state to get set. Tests should be fixed now, thanks for the catch!